decimal64.go

```diff
@@ -31,9 +31,7 @@ const (
 
 // Decimal64 represents an IEEE 754 64-bit floating point decimal number.
 // It uses the binary representation method.
-type Decimal64 struct {
-	bits uint64
-}
+type Decimal64 uint64
```

Comment:
This was intentionally done as a struct to avoid accidental casting to `uint64`, e.g.:
```go
var d Decimal64 = 23 // Oops!
```

Reply:
Hrm, I don't have an argument for or against, but I do feel that that seems to be something that needs to be documented (if it has been commented on I've missed it)

Reply:
This is more of a best practice in Go than a specific issue for decimals. Using a struct with unexported fields is the standard way to hide the internals of a type, which a bare type definition won't do.

A more philosophical objection, which kind of underlies my first point, is that a `Decimal64` isn't a `uint64` and shouldn't be defined as such.